### ISSUE TITLE
Configure Facebook Authentication Code Section

### ISSUE THREAD

**Issue by alexmackey** [aspnet/Docs#11028]
The code snippet in Configure Facebook Authentication Code, ConfigureServices could be made clearer by either including the setup of dbcontext (e.g. below) or mention this snippet should be placed after the call to AddDbContext to avoid a confusing dependency resolution error.

   services.AddDbContext&lt;ApplicationDbContext&gt;(options =&gt;
                options.UseSqlServer(
                    Configuration.GetConnectionString("DefaultConnection"))); 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f2c6faa2-8c65-073b-4988-672c1aa9690c
* Version Independent ID: d22d616f-907e-ae62-1f43-88ad176a87d4
* Content: [Facebook external login setup in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/facebook-logins?view=aspnetcore-2.2)
* Content Source: [aspnetcore/security/authentication/social/facebook-logins.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/security/authentication/social/facebook-logins.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@alexmackey The setup of `ApplicationDbContext` doesn't have anything directly to do with this tutorial. That should all be done in the previous tutorial, which is a prerequisite.

>this snippet should be placed after the call to AddDbContext to avoid a confusing dependency resolution error.

It does show that.

```
services.AddDefaultIdentity<IdentityUser>()
        .AddDefaultUI(UIFramework.Bootstrap4)
        .AddEntityFrameworkStores<ApplicationDbContext>();

services.AddAuthentication().AddFacebook(facebookOptions =>
{
    facebookOptions.AppId = Configuration["Authentication:Facebook:AppId"];
    facebookOptions.AppSecret = Configuration["Authentication:Facebook:AppSecret"];
});
``

**Comment by alexmackey** [comment]
Hey Rick I understand and maybe its just me but when I followed tutorial through I interpreted this as this code snippet replaced the existing code in configure services method which then led to the dependency resolution error (and 20 mins whilst I worked out what caused it). 

E.g. if this read "Add the Facebook service in the ConfigureServices method in the Startup.cs file below the existing call to AddDbContext" this would be clearer?
